ui/apps/platform/src/Containers/Risk/KeyValuePairs.js

```diff
@@ -72,6 +75,8 @@ class KeyValuePairs extends Component {
                     <span className={`flex-1 min-w-0 ${className}`}>
                         {isObject(value) || isArray(value) ? (
                             <div className="ml-2">{this.getNestedValue(value)}</div>
+                        ) : label === 'Deployment ID' ? (
```

Comment:
It's hard for me to say whether this condition will be hit or is actually covered by `isObject(value)`.

Reply:
How about `label === 'Deployment ID' && typeof value === 'string'`

Reply:
> It's hard for me to say whether this condition will be hit or is actually covered by `isObject(value)`.

Everything is an object in JS, except when it's not. A string is a not an object; it's one of a handful of primitives.
(Maybe you're thinking of Ruby, where even the primitives are objects 😜 )

https://github.com/getify/You-Dont-Know-JS/blob/2nd-ed/get-started/ch2.md#values

Reply:
Thank you for educating me! Please resolve once you add `&& typeof value === 'string'` as per Mark's suggestion.